<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Library Desktop Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <widget class="QLabel" name="headerLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Book Library Dashboard</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="bookListGroupBox">
      <property name="title">
       <string>Book List &amp; Search</string>
      </property>
      <layout class="QVBoxLayout" name="bookListLayout">
       <item>
        <layout class="QHBoxLayout" name="searchSortLayout">
         <item>
          <widget class="QLineEdit" name="searchInput">
           <property name="placeholderText">
            <string>Search by name or author...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="yearSearchInput">
           <property name="placeholderText">
            <string>Search by year...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="categorySearchInput">
           <property name="placeholderText">
            <string>Search by category...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="bookTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Author</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date Published</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Category</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Available</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="actionsLayout">
      <item>
       <widget class="QGroupBox" name="addBookGroupBox">
        <property name="title">
         <string>Add New Book</string>
        </property>
        <layout class="QFormLayout" name="addBookFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="addNameLabel">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="addNameInput"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="addAuthorLabel">
           <property name="text">
            <string>Author:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="addAuthorInput"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="addDatePublishedLabel">
           <property name="text">
            <string>Date Published:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="addDatePublishedInput">
           <property name="placeholderText">
            <string>e.g., 2023</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="addCategoryLabel">
           <property name="text">
            <string>Category:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="addCategoryInput"/>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="addBookButton">
           <property name="text">
            <string>Add Book</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Application Ready.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
